### PR TITLE
Contact form

### LINKED ISSUE #29: Com02 - En tant que Jacques ou Carole, je veux pouvoir contacter les développeurs du site afin  de leur faire passer mes messages (demandes, requêtes...).

### PR DESCRIPTION
**Pull request by KarenOrduna**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,11 +8,13 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "chart.js": "^2.9.4",
```
Comment (ComicScrip) [inline package.json]: what's this for ?

File: package.json [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
     "axios": "^0.21.0",
     "node-sass": "^4.14.1",
     "material-auto-rotating-carousel": "^3.0.2",
     "moment": "^2.29.1",
     "react": "^17.0.0",
+    "react-chartjs-2": "^2.10.0",
```
Comment (ComicScrip) [inline package.json]: what's this for ?

File: package.json (hunk 1 above)
Comment (ComicScrip) [reply package.json]: @KarenOrduna

File: package.json (hunk 2 above)
Comment (KarenOrduna) [reply package.json]: Ah oui, j'ai installé ça pour la graph-chart que j'etait en train de faire mais pas encore fini @ComicScrip

Comment (KarenOrduna) [reply package.json:17]: Mais c'est un autre composent donc je crois que je les ai installé dans la wrong branch? je les annule?

Comment (ComicScrip) [reply package.json]: @KarenOrduna yes you can remove them, and we'll add those deps when they are really needed :)

Comment (KarenOrduna) [reply package.json:17]: AAH I got it. C'est un package.json global! It's for a new component Graph which I haven't included yet because it's 1/3 done.

Comment (KarenOrduna) [reply package.json]: okay done :D  @ComicScrip

**Pull request merged**